config/initializers/00_extensions.rb

```diff
@@ -42,10 +42,20 @@ class FalseClass
   include Extensions::BooleanExtension::False
 end
 
-module ActionView
-  module Helpers
-    class FormBuilder
-      include Extensions::FormBuilderExtension
-    end
+# module ActionView
+#   module Helpers
+#     class FormBuilder
+#       include Extensions::FormBuilderExtension
+#     end
+#   end
+# end
```

Comment:
Cant this be removed?

Reply:
That commit it going to be removed before merging as it is only there for Rails 7.1.

Reply:
Oh, of course, I didn't spot that it was part of one of those commits 🤦